manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Google.AdsEditor
+PackageVersion: 1.3.36.152

```

```suggestion
PackageVersion: 14.2.2.0
```
Writes 14.2.2.0 to the registry